service/reducers/entities/users.js

```diff
@@ -174,6 +174,15 @@ function statuses(state = {}, action) {
     }
 }
 
+function autocompleteUsersInChannel(state = {}, action) {
```

Comment:
I'm not certain this belongs in the service layer since it's rather short-lived and view-specific. I'm 0/5 on that though. Thoughts?